ui/src/app/utils/dismissed-messages.spec.ts

```diff
@@ -0,0 +1,185 @@
+import * as cookies from './cookies';
+import {
+  getDismissedMessageIds,
+  isDismissed,
+  saveDismissedMessage,
+} from './dismissed-messages';
+
+// Mock the cookies module
+jest.mock('./cookies', () => ({
+  firstPartyCookiesEnabled: jest.fn(),
+}));
+
+describe('dismissed-messages', () => {
+  let localStorageMock: { [key: string]: string };
+  const DISMISSED_MESSAGES_KEY = 'dismissed-messages';
```

Comment:
if we export this from where it's used in the component, we can import it here instead